cli/man/splinter-circuit-disband.1.md

```diff
@@ -0,0 +1,94 @@
+% SPLINTER-CIRCUIT-DISBAND(1) Cargill, Incorporated | Splinter Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**splinter-circuit-disband** — Submits a request to disband the specified circuit.
+
+SYNOPSIS
+========
+**splinter circuit disband** \[**FLAGS**\] \[**OPTIONS**\] CIRCUIT-ID
+
+DESCRIPTION
+===========
+Request to disband a circuit by specifying the circuit ID of the circuit to be
+disbanded. Disbanding a circuit removes a circuit's networking functionality.
+Once all members of the circuit have accepted the request to disband the
+circuit, the circuit is only available offline. This functionality is currently
+behind the experimental `circuit-disband` feature and must be enabled to use
+this command.
+
+The `disband` command creates a new circuit proposal to reflect the disbanded
+state, with the proposed circuit's `circuit_status` field set to `Disbanded`.
+This proposal is then able to voted on, similar to other circuit proposals.
```

Comment:
"be" should be added between "to" and "voted"

Reply:
Updated